### PR TITLE
Remove template samples

### PR DESCRIPTION
**Pull request by jkinsky**
# Existing Sample Changes
## Description

Remove all files for 5 templates samples, one associated high-level readme.md, and the ../ProjectTemplates/.. directory and from repo in response to https://jira.devtools.intel.com/browse/ONSAM-1527. 

Fixes Issue# 

## External Dependencies

N/A

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [X] Implement fixes for ONSAM Jiras

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Command Line

### CODE REVIEW

**Comment by jkinsky** [comment]
Adding @jimmytwei for notices.

**Review by jimmytwei — APPROVED** [review]

**Comment by jimmytwei** [comment]
I'll merge once @tomlenth or @yuguen-intel approves

**Review by yuguen-intel — APPROVED** [review]

**Pull request merged**
